cylc/flow/task_remote_mgr.py

```diff
@@ -77,8 +79,10 @@ def __init__(self, suite, proc_pool):
         # self.remote_init_map = {(install target): status, ...}
         self.remote_init_map = {}
         self.uuid_str = None
+        # Ready for _what_?
```

Comment:
Happy to remove, valid comment however IMO.

Reply:
In `scheduler.py`:
```python
        if self.task_job_mgr.task_remote_mgr.ready:
             # This flag is turned on when a host init/select command completes
             process = True
             self.task_job_mgr.task_remote_mgr.ready = False  # reset
```
But yes, a comment in the remote manager class would be helpful!